src/js/compatibility.js

```diff
@@ -17,9 +17,9 @@
         Check if the version of the player requires the compatibility shim. Only versions below 8 require this script.
     */
     if (parseInt(playerLibrary.version, 10) >= 8) {
-        var jwplayerCompatible = function(query) {
```

Comment:
compatibility.js is intentionally written in ES5 for browser compatibility and should not be converted to ES6. (It is not transormed by webpack/babel).